integration-tests/cypress/commands/api/group.d.ts

```diff
@@ -1,6 +1,8 @@
 // load the global Cypress types
 /// <reference types="cypress" />
 
+import { ApiStationResponse } from "../../../../types/api/station";
```

Comment:
Pretty sure this change is going to mess with linting - but not actually break anything either way,